apisix/upstream.lua

```diff
@@ -191,7 +191,7 @@ function _M.set_by_route(route, api_ctx)
 
         local new_nodes, err = dis.nodes(up_conf.service_name)
         if not new_nodes then
-            return http_code_upstream_unavailable, "no valid upstream node: " .. (err or "nil")
+            return HTTP_CODE_UPSTREAM_UNAVAILABLE, "no valid upstream node: " .. (err or "nil")
```

Comment:
Why not use ngx.HTTP_SERVICE_UNAVAILABLE

Reply:
I think it is because the status code for this has been changed from 500 to 503. So the original contributor use a variable instead.

Reply:
OK.